src/expandProperty.js

```diff
@@ -1,12 +1,44 @@
-const WHITESPACE_NO_CALC = /\s+(?=[^)]*?(?:\(|$))/g
-const LENGTH_UNIT = /(calc\(|em|ex|ch|rem|vw|vh|vmin|vmax|cm|mm|q|in|pt|pc|px|dpi|dpcm|dppx|%|auto)$/i
```

Comment:
The `calc(` check could never have worked since the regex checks for `$` (ends with).